flight/AIs/test_linear_move.py

```diff
@@ -10,7 +10,14 @@
 controller.add_takeoff_task(f.DEFAULT_ALTITUDE)
 controller.add_hover_task(
     f.DEFAULT_ALTITUDE, HOVER_DURATION, c.Priorities.MEDIUM)
+"""
```

Comment:
This file doesn't run a linear movement

Reply:
I had accidentally left some code commented out. I have un-commented and cleaned it up.